src/explore-education-statistics-frontend/src/middleware.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import redirectPages from '@frontend/middleware/pages/redirectPages';
```

Comment:
We don't need this currently, so could remove it as we're just bundling more code than we need